gcc fails to build on OS X 10.10 (Yosemite)

```
[gcc] 		-DLIBGCC2_UNITS_PER_WORD=4
[gcc] /Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/xgcc -B/Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/bin/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/lib/ -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/include -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/sys-include    -g -O2 -m32 -O2  -g -O2 -DIN_GCC    -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-format -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition  -isystem ./include   -pipe -fno-common -g -DIN_LIBGCC2 -fbuilding-libgcc -fno-stack-protector   -pipe -fno-common -I. -I. -I../../../host-x86_64-apple-darwin14.1.0/gcc -I../../.././libgcc -I../../.././libgcc/. -I../../.././libgcc/../gcc -I../../.././libgcc/../include  -DHAVE_CC_TLS -DUSE_EMUTLS -o _fixunsdfsi.o -MT _fixunsdfsi.o -MD -MP -MF _fixunsdfsi.dep -DL_fixunsdfsi -c ../../.././libgcc/libgcc2.c -fvisibility=hidden -DHIDE_EXPORTS \
[gcc] 		-DLIBGCC2_UNITS_PER_WORD=4
[gcc] ../../.././libgcc/libgcc2.c: In function '__gcc_bcmp':
[gcc] ../../.././libgcc/libgcc2.c:2105:1: internal compiler error: Segmentation fault: 11
[gcc]  }
[gcc]  ^
[gcc] libbacktrace could not find executable to open
[gcc] Please submit a full bug report,
[gcc] with preprocessed source if appropriate.
[gcc] See <http://gcc.gnu.org/bugs.html> for instructions.
[gcc] make[5]: *** [__gcc_bcmp.o] Error 1
[gcc] /Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/xgcc -B/Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/bin/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/lib/ -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/include -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/sys-include    -g -O2 -O2  -g -O2 -DIN_GCC    -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-format -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition  -isystem ./include   -pipe -fno-common -g -DIN_LIBGCC2 -fbuilding-libgcc -fno-stack-protector   -pipe -fno-common -I. -I. -I../../host-x86_64-apple-darwin14.1.0/gcc -I../.././libgcc -I../.././libgcc/. -I../.././libgcc/../gcc -I../.././libgcc/../include  -DHAVE_CC_TLS -DUSE_EMUTLS -o _fixunsxfsi_s.o -MT _fixunsxfsi_s.o -MD -MP -MF _fixunsxfsi_s.dep -DSHARED -DL_fixunsxfsi -c ../.././libgcc/libgcc2.c \
[gcc] 		-DLIBGCC2_UNITS_PER_WORD=4
[gcc] make[5]: *** Waiting for unfinished jobs....
[gcc] /Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/xgcc -B/Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/bin/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/lib/ -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/include -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/sys-include    -g -O2 -O2  -g -O2 -DIN_GCC    -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-format -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition  -isystem ./include   -pipe -fno-common -g -DIN_LIBGCC2 -fbuilding-libgcc -fno-stack-protector   -pipe -fno-common -I. -I. -I../../host-x86_64-apple-darwin14.1.0/gcc -I../.././libgcc -I../.././libgcc/. -I../.././libgcc/../gcc -I../.././libgcc/../include  -DHAVE_CC_TLS -DUSE_EMUTLS -o _fixsfdi_s.o -MT _fixsfdi_s.o -MD -MP -MF _fixsfdi_s.dep -DSHARED -DL_fixsfdi -c ../.././libgcc/libgcc2.c \
[gcc] 		-DLIBGCC2_UNITS_PER_WORD=4
[gcc] /Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/xgcc -B/Users/certik/.hashdist/tmp/gcc-l7u34nnvgab4/host-x86_64-apple-darwin14.1.0/gcc/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/bin/ -B/Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/lib/ -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/include -isystem /Users/certik/.hashdist/bld/gcc/l7u34nnvgab4/x86_64-apple-darwin14.1.0/sys-include    -g -O2 -O2  -g -O2 -DIN_GCC    -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-format -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition  -isystem ./include   -pipe -fno-common -g -DIN_LIBGCC2 -fbuilding-libgcc -fno-stack-protector   -pipe -fno-common -I. -I. -I../../host-x86_64-apple-darwin14.1.0/gcc -I../.././libgcc -I../.././libgcc/. -I../.././libgcc/../gcc -I../.././libgcc/../include  -DHAVE_CC_TLS -DUSE_EMUTLS -o _fixdfdi_s.o -MT _fixdfdi_s.o -MD -MP -MF _fixdfdi_s.dep -DSHARED -DL_fixdfdi -c ../.././libgcc/libgcc2.c \
[gcc] 		-DLIBGCC2_UNITS_PER_WORD=4
[gcc] make[4]: *** [multi-do] Error 1
[gcc] make[3]: *** [all-multi] Error 2
[gcc] make[3]: *** Waiting for unfinished jobs....
[gcc] make[2]: *** [all-stage1-target-libgcc] Error 2
[gcc] make[1]: *** [stage1-bubble] Error 2
[gcc] make: *** [all] Error 2
[gcc|ERROR] Command '[u'/bin/bash', '_hashdist/build.sh']' returned non-zero exit status 2
[gcc|ERROR] command failed (code=2); raising

real	8m33.220s
user	10m56.568s
sys	4m12.526s
```
Full build log available [here](https://gist.githubusercontent.com/certik/6530d081a92c9b28a8ed/raw/build.log).
I am using 
```
$ clang -v
Apple LLVM version 6.0 (clang-600.0.56) (based on LLVM 3.5svn)
Target: x86_64-apple-darwin14.1.0
Thread model: posix
```
To compile it and I am using the profile https://gist.github.com/certik/3175503c8e1176771423 with just `gcc` added to it.

I looks like the error is coming from the initial gcc compiler build by clang, i.e. it's coming from gcc, not from clang.

I am building GCC on OS X Yosemite, version 10.10.2, with XCode version 6.3.1.

My default.yaml profile looks like this:

```
extends:
- file: osx_clt.yaml

parameters:
  PROLOGUE: |
    export CC=/Applications/Xcode.app/Contents/Developer/usr/bin/gcc; export CXX=/Applications/Xcode.app/Contents/Developer/usr/bin/g++;

packages:
  gcc:
    sources:
    - url: http://ftpmirror.gnu.org/gcc/gcc-4.9.2/gcc-4.9.2.tar.bz2
      key: tar.bz2:eaqmtauvqvvkcp62b4xtur4ujedvp7be
  python:
    host: true
```

 `gcc` and `g++` are versions:

```
/Applications/Xcode.app/Contents/Developer/usr/bin/gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 6.1.0 (clang-602.0.49) (based on LLVM 3.6.0svn)
Target: x86_64-apple-darwin14.1.0
Thread model: posix
```
```
/Applications/Xcode.app/Contents/Developer/usr/bin/g++ --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 6.1.0 (clang-602.0.49) (based on LLVM 3.6.0svn)
Target: x86_64-apple-darwin14.1.0
Thread model: posix
```

When running HashDist, I get the following error:

```
2015/04/29 15:06:40 - INFO: [package:run_job] clang: warning: treating 'c' input as 'c++' when in C++ mode, this behavior is deprecated
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from ../.././gcc/graphite.c:38:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/set.h:13:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/map_type.h:4:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/ctx.h:16:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/int.h:15:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/gmp/e7miyy3xe6ci/include/gmp.h:51:
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/cstddef:46:9: error: no member named 'ptrdiff_t' in the global namespace
2015/04/29 15:06:40 - INFO: [package:run_job] using ::ptrdiff_t;
2015/04/29 15:06:40 - INFO: [package:run_job]       ~~^
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from ../.././gcc/graphite.c:38:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/set.h:13:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/map_type.h:4:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/ctx.h:16:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/int.h:17:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iostream:38:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ios:216:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__locale:15:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/string:439:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:628:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:604:
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iterator:402:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:40 - INFO: [package:run_job]     typedef ptrdiff_t difference_type;
2015/04/29 15:06:40 - INFO: [package:run_job]             ^
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iterator:429:56: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:40 - INFO: [package:run_job] template<class _Category, class _Tp, class _Distance = ptrdiff_t,
2015/04/29 15:06:40 - INFO: [package:run_job]                                                        ^
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iterator:745:66: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:40 - INFO: [package:run_job]           class _Traits = char_traits<_CharT>, class _Distance = ptrdiff_t>
2015/04/29 15:06:40 - INFO: [package:run_job]                                                                  ^
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iterator:745:54: error: template parameter missing a default argument
2015/04/29 15:06:40 - INFO: [package:run_job]           class _Traits = char_traits<_CharT>, class _Distance = ptrdiff_t>
2015/04/29 15:06:40 - INFO: [package:run_job]                                                      ^
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iterator:745:27: note: previous default template argument defined here
2015/04/29 15:06:40 - INFO: [package:run_job]           class _Traits = char_traits<_CharT>, class _Distance = ptrdiff_t>
2015/04/29 15:06:40 - INFO: [package:run_job]                           ^
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from ../.././gcc/graphite.c:38:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/set.h:13:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/map_type.h:4:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/ctx.h:16:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Users/lindseybugbee/.hashdist/bld/isl/6snfizl7aufi/include/isl/int.h:17:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iostream:38:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ios:216:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__locale:15:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/string:439:
2015/04/29 15:06:40 - INFO: [package:run_job] In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:628:
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:755:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:40 - INFO: [package:run_job]     typedef ptrdiff_t type;
2015/04/29 15:06:40 - INFO: [package:run_job]             ^
2015/04/29 15:06:40 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:903:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:40 - INFO: [package:run_job]     typedef ptrdiff_t difference_type;
2015/04/29 15:06:40 - INFO: [package:run_job]             ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1522:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]             ptrdiff_t _Np = __end1 - __begin1;
2015/04/29 15:06:41 - INFO: [package:run_job]             ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1552:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]             ptrdiff_t _Np = __end1 - __begin1;
2015/04/29 15:06:41 - INFO: [package:run_job]             ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1616:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]     typedef ptrdiff_t         difference_type;
2015/04/29 15:06:41 - INFO: [package:run_job]             ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1708:13: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]     typedef ptrdiff_t         difference_type;
2015/04/29 15:06:41 - INFO: [package:run_job]             ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1805:23: error: use of undeclared identifier 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]                       ptrdiff_t,                                   // purposefully not C++03
2015/04/29 15:06:41 - INFO: [package:run_job]                       ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1807:65: error: expected class name
2015/04/29 15:06:41 - INFO: [package:run_job]                       raw_storage_iterator<_OutputIterator, _Tp>&> // purposefully not C++03
2015/04/29 15:06:41 - INFO: [package:run_job]                                                                 ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1822:12: error: use of undeclared identifier 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job] pair<_Tp*, ptrdiff_t>
2015/04/29 15:06:41 - INFO: [package:run_job]            ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1823:22: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job] get_temporary_buffer(ptrdiff_t __n) _NOEXCEPT
2015/04/29 15:06:41 - INFO: [package:run_job]                      ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1825:16: error: use of undeclared identifier 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]     pair<_Tp*, ptrdiff_t> __r(0, 0);
2015/04/29 15:06:41 - INFO: [package:run_job]                ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1826:11: error: unknown type name 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]     const ptrdiff_t __m = (~ptrdiff_t(0) ^
2015/04/29 15:06:41 - INFO: [package:run_job]           ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1826:29: error: use of undeclared identifier 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]     const ptrdiff_t __m = (~ptrdiff_t(0) ^
2015/04/29 15:06:41 - INFO: [package:run_job]                             ^
2015/04/29 15:06:41 - INFO: [package:run_job] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/memory:1827:38: error: use of undeclared identifier 'ptrdiff_t'
2015/04/29 15:06:41 - INFO: [package:run_job]                            ptrdiff_t(ptrdiff_t(1) << (sizeof(ptrdiff_t) * __CHAR_BIT__ - 1)))
2015/04/29 15:06:41 - INFO: [package:run_job]                                      ^
2015/04/29 15:06:41 - INFO: [package:run_job] fatal error: too many errors emitted, stopping now [-ferror-limit=]
2015/04/29 15:06:45 - INFO: [package:run_job] 20 errors generated.
2015/04/29 15:06:45 - INFO: [package:run_job] make[3]: *** [graphite.o] Error 1
2015/04/29 15:06:45 - INFO: [package:run_job] make[2]: *** [all-stage1-gcc] Error 2
2015/04/29 15:06:45 - INFO: [package:run_job] make[1]: *** [stage1-bubble] Error 2
2015/04/29 15:06:45 - INFO: [package:run_job] make: *** [all] Error 2
2015/04/29 15:06:45 - ERROR: [package:run_job] Command '[u'/bin/bash', '_hashdist/build.sh']' returned non-zero exit status 2
2015/04/29 15:06:45 - ERROR: [package:run_job] command failed (code=2); raising
```

I tried modifying the PROLOGUE to the following to suppress the error limit:
```
parameters:
  PROLOGUE: |
    export CC=clang; export CXX=clang++; alias clang="clang -ferror-limit=0"; alias clang++="clang++ -ferror-limit=0"
```
But that did not change the error.

I was only able to install GCC from source using Homebrew. Here is their recipe:
https://github.com/Homebrew/homebrew/blob/master/Library/Formula/gcc.rb

--- a/pkgs/gmp.yaml
+++ b/pkgs/gmp.yaml
@@ -4,8 +4,8 @@ dependencies:
   build: [m4]
 
 sources:
-- url: http://mirrors.kernel.org/gnu/gmp/gmp-5.1.3.tar.bz2
-  key: tar.bz2:ouqhsuqli2ifgelr2d2fglsa6cdaaik7
+- key: tar.bz2:p6hjvacltrwqofsm65kca67ihdwocimu
+  url: http://mirrors.kernel.org/gnu/gmp/gmp-6.0.0a.tar.bz2
 
 build_stages:
 - name: configure

--- a/pkgs/gmp/gmp.yaml
+++ b/pkgs/gmp/gmp.yaml
@@ -8,6 +8,12 @@ sources:
   url: http://mirrors.kernel.org/gnu/gmp/gmp-6.0.0a.tar.bz2
 
 build_stages:
+- name: rpath_patch
+  before: configure
+  files: [osx_10_10_fix.patch]
+  handler: bash
+  bash: |
+    patch -up1 < _hashdist/osx_10_10_fix.patch
 - name: configure
   extra: ['--enable-cxx']
 

--- a/pkgs/gmp/osx_10_10_fix.patch
+++ b/pkgs/gmp/osx_10_10_fix.patch
@@ -1,0 +1,34 @@
+commit 4519b66df0c8a175270c2caf4338c44159e594cb
+Author: Ondřej Čertík <ondrej.certik@gmail.com>
+Date:   Fri May 1 00:08:15 2015 -0600
+
+    Make configure scripts detect OS X 10.10 properly
+    
+    Previously it treated 10.10 as 10.1.
+
+diff --git a/aclocal.m4 b/aclocal.m4
+index 11222f5..37c1162 100644
+--- a/aclocal.m4
++++ b/aclocal.m4
+@@ -1066,7 +1066,7 @@ _LT_EOF
+       case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
+ 	10.0,*86*-darwin8*|10.0,*-darwin[[91]]*)
+ 	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;
+-	10.[[012]]*)
++	10.[[012]][[,.]]*)
+ 	  _lt_dar_allow_undefined='${wl}-flat_namespace ${wl}-undefined ${wl}suppress' ;;
+ 	10.*)
+ 	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;
+diff --git a/configure b/configure
+index 60c6bd5..b223c18 100755
+--- a/configure
++++ b/configure
+@@ -14809,7 +14809,7 @@ $as_echo "$lt_cv_ld_force_load" >&6; }
+       case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
+ 	10.0,*86*-darwin8*|10.0,*-darwin[91]*)
+ 	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;
+-	10.[012]*)
++	10.[012][,.]*)
+ 	  _lt_dar_allow_undefined='${wl}-flat_namespace ${wl}-undefined ${wl}suppress' ;;
+ 	10.*)
+ 	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;

